src/ol_openedx_chat/block.py

```diff
@@ -66,23 +72,51 @@ def student_view_aside(self, block, context=None):
         if not self.ol_chat_enabled:
             return fragment
 
+        block_id = self.scope_ids.usage_id.usage_key.block_id
+        block_usage_key = self.scope_ids.usage_id.usage_key
+        block_type = getattr(block, "category", None)
+
         fragment.add_content(
             render_template(
                 "static/html/student_view.html",
                 {
-                    "block_key": self.scope_ids.usage_id.usage_key.block_id,
-                    "block_type": getattr(block, "category", None),
+                    "block_id": block_id,
+                    "block_usage_key": block_usage_key,
+                    "block_type": block_type,
                 },
             )
         )
         fragment.add_css(get_resource_bytes("static/css/ai_chat.css"))
         fragment.add_javascript(get_resource_bytes("static/js/ai_chat.js"))
+
+        request_opts = {
+            "block_id": block_id,
+            "block_usage_key": block_usage_key,
+        }
+
+        if block_type == VIDEO_BLOCK_CATEGORY:
+            try:
+                transcripts_info = block.get_transcripts_info()
+                if transcripts_info.get("transcripts") and transcripts_info[
+                    "transcripts"
+                ].get("en"):
+                    request_opts["transcript_asset_id"] = Transcript.asset_location(
+                        block.location,
+                        transcripts_info["transcripts"][ENGLISH_LANGUAGE_TRANSCRIPT],
+                    )
+
+            except Exception:
```

Comment:
It's possible that the video might not have any transcripts, or no English language transcript. For example, it might be a demonstration video without audio. 

Warning on missing transcripts should be OK, but I don't think we want to raise an exception.